### PR TITLE
Show exit code for failed actions

### PR DESCRIPTION
**Pull request by luluz66**
Pass action result digest as a url when digest is clicked.
Use action result digest when fetch action result if available.
The FE behavior doesn't change when the BE PR https://github.com/buildbuddy-io/buildbuddy/pull/1332 is not there;
With the BE PR, it shows results details for the failed actions. (It doesn't include stderr/stdout in this PR) 

Screenshot
Screen Shot 2021-12-14 at 5 57 01 PM

### CODE REVIEW

**Review by siggisim — APPROVED** [review]

**Comment by siggisim** [comment]
This is so awesome - been wanting this one for a long time!

**Comment by brentleyjones** [comment]
Woot!

File: app/invocation/invocation_action_card.tsx [hunk 1]
```diff
@@ -74,7 +74,11 @@ export default class InvocationActionCardComponent extends React.Component<Props
   }
 
   fetchActionResult() {
-    const digest = parseDigest(this.props.search.get("actionDigest"));
+	let digestParam = this.props.search.get("actionResultDigest");
+	if (digestParam == null) {
+	    digestParam = this.props.search.get("actionDigest");
+	}
```
Comment (bduffany) [inline app/invocation/invocation_action_card.tsx:80]: Looks like this needs formatting; can you run `./buildfix.sh` to fix it?

**Review by bduffany — CHANGES_REQUESTED** [review]

File: app/invocation/invocation_action_card.tsx (hunk 1 above)
Comment (luluz66) [reply app/invocation/invocation_action_card.tsx]: Done.

**Review by bduffany — APPROVED** [review]

**Pull request merged**